docs/docs/standard-lib/sqlite.md

```diff
@@ -25,7 +25,7 @@ Note: Unlike SQLite and most other libraries, foreign keys **are** enabled by de
 import Sqlite;
 ```
 
-### Sqlite.connect(string: database)
+### Sqlite.connect(String: database) -> Result
```

Comment:
```suggestion
### Sqlite.connect(String: database) -> Result<SQLite>
```

What do you think about displaying the value a Result type wraps too?

Reply:
I dig it. I'll update the other return types to this format for `Result`.